### ISSUE TITLE
AttributeError: module 'fluent.asynchandler' has no attribute 'FluentRecordFormatter'

### ISSUE THREAD

**Issue by jiuchongxiao** [fluent/fluent-logger-python#140]
asyn loging has erro
AttributeError: module 'fluent.asynchandler' has no attribute 'FluentRecordFormatter'
detail:
python:3.6.5
pip:10.0.1
![default](https://user-images.githubusercontent.com/13093730/40523111-bb3d9f26-6006-11e8-9c41-d1ef207295a6.png)

**Comment by arcivanov** [comment]
It's not supposed to. https://github.com/fluent/fluent-logger-python/blob/master/fluent/handler.py#L20

**Comment by jiuchongxiao** [comment]
@arcivanov  sorry,may i did not say clear,  i say asynchandler.py

**Comment by arcivanov** [comment]
Yes, you did, no FluentRecordFormatter resides in the handler, not asynchandler.

**Comment by mheppner** [comment]
There's confusion with this because the async section of the [README](https://github.com/fluent/fluent-logger-python/blob/10fe47c7fd56d5dd0e6ca63cca2c00450a9b4415/README.rst#asynchronous-communication) is importing `from fluent import asynchandler as handler` and using `handler.FluentRecordFormatter()`.

**Comment by arcivanov** [comment]
Thanks, I'll take a look at it.

**Comment by chikinchoi** [comment]
@arcivanov 
still cannot find the FluentRecordFormatter Class in asynchandler.  Could you update the Readme and show how to add the custom formatter to the asynchandler? Thank you

**Comment by SebastianScherer88** [comment]
@chikinchoi you need to import [the `FluentRecordFormatter` class from the `handler.py`](class FluentRecordFormatter(logging.Formatter, object):) - the README is just wrong on that bit. the formatter is just formatting logs, not outputting, so no reason to implement (or import) some sort of "async formatter".

tl;dr - the dos are wrong
